standalone/pherry/src/lib.rs

```diff
@@ -1240,9 +1270,10 @@ async fn bridge(
 
         // check if pRuntime has already reached the chain tip.
         if synced_blocks == 0 && !more_blocks {
-            if !initial_sync_finished && !args.no_register && !flags.worker_registered {
-                try_register_worker(&pr, &para_api, &mut signer, operator.clone(), args).await?;
-                flags.worker_registered = true;
+            if !args.no_register && !flags.worker_registered {
```

Comment:
Now the pruntime will not emit the RA report until the genesis state has been validated.
So, it may not actually send the register after the `initial_sync_finished` is set to true.